actions/inventory.go

```diff
@@ -50,7 +50,7 @@ func InitCollectors(trace bool) *Collectors {
 //
 // The lshw collector always executes first and is included by default.
 // nolint:gocyclo //since we're collecting inventory for each type, this is cyclomatic
-func Collect(ctx context.Context, device *common.Device, collectors *Collectors, trace, failOnError bool) error {
+func Collect(ctx context.Context, device *common.Device, collectors *Collectors, trace, failOnError bool, dynamic bool) error {
```

Comment:
suggestion - the bool parameter type definition can be collapsed
```
func Collect(ctx context.Context, device *common.Device, collectors *Collectors, trace, failOnError, dynamic bool)
```

going ahead, I think this should use the options pattern https://github.com/tmrts/go-patterns/blob/master/idiom/functional-options.md